pre-commit/static_tests.sh

```diff
@@ -56,5 +56,5 @@ docker run --rm \
 	-v "$SCRIPT_DIR":/pre-commit/ \
 	-w /src \
 	"$IMAGE" /bin/bash -c \
-	"/opt/python/cp39-cp39/bin/pip install jsonschema lxml
+	"/opt/python/cp39-cp39/bin/pip install jsonschema lxml django
```

Comment:
Probably need to keep this in sync with SOAR:
```suggestion
	"/opt/python/cp39-cp39/bin/pip install jsonschema lxml django<5
```